tabular/src/autogluon/tabular/features/generators/auto_ml_pipeline.py

```diff
@@ -75,14 +80,19 @@ def __init__(self, enable_numeric_features=True, enable_categorical_features=Tru
         self.enable_datetime_features = enable_datetime_features
         self.enable_text_special_features = enable_text_special_features
         self.enable_text_ngram_features = enable_text_ngram_features
+        self.enable_raw_text_features = enable_raw_text_features
 
         generators = self._get_default_generators(vectorizer=vectorizer)
         super().__init__(generators=generators, **kwargs)
 
     def _get_default_generators(self, vectorizer=None):
         generator_group = []
         if self.enable_numeric_features:
-            generator_group.append(IdentityFeatureGenerator(infer_features_in_args=dict(valid_raw_types=[R_INT, R_FLOAT])))
+            generator_group.append(IdentityFeatureGenerator(infer_features_in_args=dict(
+                valid_raw_types=[R_INT, R_FLOAT])))
+        if self.enable_raw_text_features:
+            generator_group.append(IdentityFeatureGenerator(infer_features_in_args=dict(
+                valid_special_types=[S_TEXT]), name_suffix='_raw_text'))
```

Comment:
`valid_special_types` -> `required_special_types` , otherwise it will not filter out features with no special types.